mycroft/client/enclosure/mark1/__init__.py

```diff
@@ -166,7 +166,7 @@ def process(self, data):
             self.bus.emit(Message("speak", {
                 'utterance': mycroft.dialog.get("reset to factory defaults")}))
             subprocess.call(
-                (f'rm {xdg.BaseDirectory.save_config_path("mycroft")}'
+                (f'rm {xdg.BaseDirectory.xdg_config_home("mycroft")}'
```

Comment:
missing a join ?

Reply:
Indeed!

Reply:
And the previous path didn't look right either...but should be better now